src/components/Status/Status.js

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+class Status extends Component {
+
+  render () {
+    const { success, error } = this.props;
+
+    if (success) {
+      return (
+        <p className="alert alert-success">{success}</p>
+      );
+    }
+
+    if (error) {
+      return (
+        <p className="alert alert-danger">{error}</p>
+      );
+    }
+
+    return false;
+  }
+}
+
+const mapStateToProps = (state) => {
+  if (state.msg) {
```

Comment:
would like to see a test on this - basically it just tests to make sure that the right values are sent back from state. started running into problems here - maybe around local storage? unknown.